docs/admin/external-auth.md

```diff
@@ -1,5 +1,10 @@
 # External Authentication
 
+Coder supports external authentication via OAuth2.0. This allows enabling any OAuth provider as well as integrations with Git providers, such as GitHub, GitLab, and Bitbucket.
+
+External authentication can also be used to integrate with external services
+like JFrog Artifactory and others.
+
 To add an external authentication provider, you'll need to create an OAuth
 application. The following providers are supported:
```

Comment:
I would change it to say "we have tested the following providers but others should work too"